src/js/background.js

```diff
@@ -18,3 +18,46 @@ chrome.extension.onConnect.addListener(function(port) {
     });
 });
 
+/**
+ * Holds the state of debug mode. True means we should debug the page.
+ */
+let debug = false;
+let hasAPIs = chrome.webRequest;
+
+/**
+ * Responds to clicks on the extension's icon. Toggles debug mode.
+ */
+chrome.browserAction.onClicked.addListener((tab) => {
+
+    debug = !debug;
+    chrome.browserAction.setTitle({
+        title: debug ? 'Sidecar Debug: ON' : 'Sidecar Debug: OFF'
+    });
+    // chrome.browserAction.setIcon({
+    //     path: debug ? 'icon-on.png' : 'icon-off.png'
+    // });
+    chrome.browserAction.setBadgeText({
+        text: debug ? 'ON' : ''
+    });
+
+    chrome.tabs.update(tab.id, {url: tab.url, selected: tab.selected}, null);
+
+    // webRequest handlers changed (solves caching issues)
+    hasAPIs && chrome.webRequest.handlerBehaviorChanged();
+});
+
+hasAPIs && chrome.webRequest.onCompleted.addListener((details) => {
+
+    if (!debug) {
+        return;
+    }
+
+    if (details.url.indexOf('sidecar.min.js.map') > 0) {
+        return;
```

Comment:
cuz on a few lines below we are filtering using the `sidecar.min.js` or `sidecar.js`, but match patterns (https://developer.chrome.com/extensions/match_patterns) aren't good enough for us to say that we don't want the map (since those 2 files have urls like `?v=to_avoid_cache`) I had to put the extra `*` at the end of the file